cmd/semdiffstat/main.go

```diff
@@ -49,22 +50,23 @@ func main() {
 		log.Fatalf("could not parse Go files %v and %v: %v\n", aName, bName, err)
 	}
 
+	w := new(tabwriter.Writer)
+	w.Init(os.Stdout, 0, 8, 0, '\t', 0)
```

Comment:
Did you test this? I suspect you want something more like: `(os.Stdout, 0, 0, 0, ' ', 0)`

Reply:
Thanks, GitHub, for a highly questionable line break decision here. The fifth parameter suggested is a space (`" "`).

Reply:
Done. I should I've read the docs better (it looked good on my screen though)